app/src/main/java/com/codesoom/assignment/MyHttpHandler.java

```diff
@@ -0,0 +1,72 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyHttpHandler implements HttpHandler {
+
+    JSONConverter jsonConverter = new JSONConverter();
+
+    private List<Task> tasks = new ArrayList<>();
+
+    public MyHttpHandler() {
+        Task task = new Task();
+        task.setId(1L);
+        task.setTitle("Task1");
+
+        tasks.add(task);
+    }
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+
+        HttpRequestInfo httpRequestInfo = new HttpRequestInfo(exchange);
+
+        if (!httpRequestInfo.getBody().isBlank()) {
+            // JSON 데이터를 task로 변환
+            Task task = jsonConverter.JSONToTask(httpRequestInfo.getBody());
+            tasks.add(task);
+        }
+
+        String content = "Hello, world!";
+
+        if (!httpRequestInfo.getMethod().isEmpty()) {
+            switch (httpRequestInfo.getMethod()) {
+                case "GET":
+                    System.out.println(httpRequestInfo.toString());
+                    break;
+                case "POST":
+                    System.out.println(httpRequestInfo.toString());
+                    break;
+                case "PUT":
+                    System.out.println(httpRequestInfo.toString());
+                    break;
+                case "PATH":
+                    System.out.println(httpRequestInfo.toString());
+                    break;
+                case "DELETE":
+                    System.out.println(httpRequestInfo.toString());
+                    break;
+                default:
+                    System.out.println(httpRequestInfo.toString());
+                    break;
+            }
+            exchange.sendResponseHeaders(200, content.length());
+        } else {
+            content = "There isn't Method";
+            exchange.sendResponseHeaders(404, content.length());
+        }
+
+        // response 처리
+        OutputStream outputStream = exchange.getResponseBody(); // getResponseBody() : Response를 byte 배열로 반환
+        outputStream.write(content.getBytes());  // 매개값으로 주어진 바이트 배열의 모든 바이트를 출력 스트림으로 보냄
+        outputStream.flush(); // 버퍼에 남아있는 데이터를 모두 출력시키고 버퍼를 비움
+        outputStream.close(); // 호출해서 사용했던 시스템 자원을 풀어줌
+    }
+}
```

Comment:
코드 마지막 줄에는 빈 줄이 하나가 추가되어야 합니다. 그렇지 않으면 위와 같이 GitHub에서 표시가 되고 또한 eslint에서 통과하지 못합니다. 

![image](https://user-images.githubusercontent.com/14071105/105195149-adcf0f80-5b7d-11eb-9cd1-38abf1e49195.png)
```suggestion
}

// 블라블라
```
이 것이 문제가 되는 이유는 POSIX 표준에서 Line을 “개행 문자로 끝나는 0개 이상의 개행 문자가 아닌 문자들의 시퀀스”으로 정의하고 있기 때문입니다.

```js
console.log('Hello');
```
만약 위와 같이 빈 줄이 없는 코드에서 아래에 코드를 추가하게 되면 Git은 다음과 같이 표시합니다.

```js
- console.log('Hello');
+ console.log('Hello');
+ console.log('world!');
```
저는 단순히 아래 한 줄만 추가했는데 왜 세줄이 변경되었다고 나오는 걸까요? 그 이유는 줄바꿈 문자 \n가 들어갔기 때문입니다. 

```js
- console.log('Hello');
+ console.log('Hello');\n
+ console.log('world');
```
만약 미리 빈 줄을 하나 추가한 상태에서 아래에 새로운 코드를 작성할 경우에는 다음과 같이 보여집니다.

```js
+ console.log('world');
```

이미 \n이 추가되어 있기 때문이죠. 따라서 코드 마지막 줄에 빈 줄 하나를 항상 추가해주는게 좋습니다.

Reply:
넵 알겠습니다!